### PR TITLE
Validate received content

### PR DESCRIPTION
**Pull request by perama-v**
### Issue addressed

- Progress towards #96 

### Description

When performing an audit, if content is received, attempt to 
- Check the structure of the content is correct (encoding)
- The second part of the check is addressed in 
    - #100 

   
### Other changes

Cargo.lock: Updated to latest ethportal-api via `cargo update -p ethportal-api` to get HistoryContentValue. Corresponding
changes to import paths for existing ethportal-api.

### CODE REVIEW

File: entity/src/content.rs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 use anyhow::Result;
 use chrono::{DateTime, FixedOffset, Utc};
 use ethereum_types::H256;
-use ethportal_api::types::content_key::OverlayContentKey;
```
Comment (pipermerriam) [inline entity/src/content.rs:5]: will you pull these changes out into a separate PR so that we can minimize the diff sizes being reviewed?

**Review by pipermerriam — APPROVED** [review]
Small dependency rebase/fix to be done and I'll get this merged

**Comment by perama-v** [comment]
> Small dependency rebase/fix to be done and I'll get this merged

Updated. I also swapped `glados-cartographer` to use `ethportal-api` via git to get 0.2.0 (crates.io is lagging)
